payments-core/detekt-baseline.xml

```diff
@@ -25,6 +24,7 @@
     <ID>EmptyFunctionBlock:PaymentSessionViewModel.kt$PaymentSessionViewModel${ }</ID>
     <ID>EmptyFunctionBlock:StripeTextWatcher.kt$StripeTextWatcher${ }</ID>
     <ID>ExplicitGarbageCollectionCall:WeakMapInjectorRegistryTest.kt$WeakMapInjectorRegistryTest$gc()</ID>
+    <ID>ImportOrdering:com.stripe.android.googlepaylauncher.GooglePayLauncherViewModelTest.kt:3</ID>
```

Comment:
Can we fix this?

Reply:
🤦 I fixed the ordering but forgot to update the detekt baseline. will do.